engine/runtime/src/test/java/org/enso/compiler/EnsoCompilerTest.java

```diff
@@ -138,15 +138,6 @@ public void testIsDigitWithSpaces() throws Exception {
     """);
   }
 
-  @Test
-  @Ignore
-  public void testIsDigitWithoutSpaces() throws Exception {
```

Comment:
Right, this test is misleading as Wojciech wrote:

> `is_digit=character -> 42` is just `(is_digit=character) -> 42`, which is a lambda with named argument

the old parser doesn't understand that.